### PR TITLE
APR-2838 admin service remove code that was migrated to fha repo

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

**Review by MarkFCain — DISMISSED** [review]

File: src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs [hunk 1]
```diff
@@ -21,46 +21,12 @@ public RoatpApplicationApiClient(string baseUri, ILogger<RoatpApplicationApiClie
         {
         }
 
+        //TODO: Will be removed as part of snapshot code cleanup APR-2975
```
Comment (DevenKShah) [inline src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs:24]: Delete the TODO comment

File: src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs [hunk 2]
```diff
@@ -530,10 +530,8 @@ private async Task GatherOrganisationStandards(RegisterViewAndEditOrganisationVi
                 viewAndEditModel.RegisterViewOrganisationStandardsViewModel.OrganisationStandards.PaginationViewModel = await GatherOrganisationStandards(viewAndEditModel.OrganisationId, paged);
                 
                 viewAndEditModel.RegisterViewOrganisationStandardsViewModel.InProgressApplications.OrganisationId = viewAndEditModel.OrganisationId;
-                viewAndEditModel.RegisterViewOrganisationStandardsViewModel.InProgressApplications.PaginationViewModel = await GatherOrganisationStandardApplications(viewAndEditModel.OrganisationId, ApplicationReviewStatus.InProgress);
```
Comment (DevenKShah) [inline src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs:533]: Should we not be removing the properties from RegisterViewOrganisationStandardsViewModel?

**Review by DevenKShah — APPROVED** [review]
Looks good. Get rid of todo comments.

File: src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs (hunk 1 above)
Comment (raviudari01) [reply src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs]: TODO comment is removed left a pointer comment for APR-2975

File: src/SFA.DAS.AssessorService.ApplyTypes/Roatp/Apply/Apply.cs [hunk 3]
```diff
@@ -5,6 +5,7 @@
 namespace SFA.DAS.AssessorService.ApplyTypes.Roatp.Apply
 {
     //TODO: Tweak after Roatp FHA migration (APR-1823)
+    //TODO: Will be removed as part of snapshot code cleanup APR-2975
```
[the remark this answers is not in the document]
Comment (raviudari01) [reply src/SFA.DAS.AssessorService.ApplyTypes/Roatp/Apply/Apply.cs]: TODO comment is removed left a pointer comment for APR-2975

File: src/SFA.DAS.AssessorService.ApplyTypes/Roatp/Apply/Apply.cs [hunk 4]
```diff
@@ -27,76 +28,25 @@ public class RoatpApply
...
     public class RoatpApplyData
     {
         public List<RoatpApplySequence> Sequences { get; set; }
         public RoatpApplyDetails ApplyDetails { get; set; }
         public RoatpApplyGatewayDetails GatewayReviewDetails { get; set; }
     }
 
+    //TODO: Remove after Roatp FHA migration (APR-1823)
+    //TODO: Will be removed as part of snapshot code cleanup APR-2975
```
[the remark this answers is not in the document]
Comment (raviudari01) [reply src/SFA.DAS.AssessorService.ApplyTypes/Roatp/Apply/Apply.cs]: TODO comment is removed left a pointer comment for APR-2975

File: src/SFA.DAS.AssessorService.ApplyTypes/Roatp/Apply/Apply.cs [hunk 5]
```diff
@@ -27,76 +28,25 @@ public class RoatpApply
...
-
-    //TODO: Remove after Roatp FHA migration (APR-1823)
-    public static class FinancialApplicationOutcome
-    {
-        public const string Passed = "Passed";
-        public const string Failed = "Failed";
-    }
-
     //TODO: Tweak after Roatp FHA migration (APR-1823)
+    //TODO: Will be removed as part of snapshot code cleanup APR-2975
```
[the remark this answers is not in the document]
Comment (raviudari01) [reply src/SFA.DAS.AssessorService.ApplyTypes/Roatp/Apply/Apply.cs]: TODO comment is removed left a pointer comment for APR-2975

File: src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs [hunk 6]
```diff
@@ -570,34 +568,6 @@ private async Task<PaginationViewModel<OrganisationStandardSummary>> GatherOrgan
             return null;
         }
 
-        private async Task<PaginationViewModel<ApplicationSummaryItem>> GatherOrganisationStandardApplications(string organisationId, string reviewStatus)
-        {
-            var standardApplicationsRequest = new StandardApplicationsRequest
```
[the remark this answers is not in the document]
Comment (raviudari01) [reply src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs:575]: StandardApplicationsRequest is used for EPAO functionality

File: src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs (hunk 2 above)
Comment (raviudari01) [reply src/SFA.DAS.AdminService.Web/Controllers/RegisterController.cs:533]: RegisterViewOrganisationStandardsViewModel is in use, GatherOrganisationStandardApplications method call is restored

**Review by DevenKShah — DISMISSED** [review]

**Review by MarkFCain — APPROVED** [review]

**Review by CodeWolf86 — APPROVED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**